tests/clone_test.go

```diff
@@ -136,7 +136,7 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 		return snapshot
 	}
 
-	generateCloneFromVMWithParams := func(sourceVM *virtv1.VirtualMachine, targetVMName string) *clonev1alpha1.VirtualMachineClone {
+	generateCloneFromVMWithParams := func(sourceVM *virtv1.VirtualMachine, targetVMName, hostName string) *clonev1alpha1.VirtualMachineClone {
```

Comment:
Please avoid adding the hostname as a parameter here and define it in the specific test that's relevant